tutorials/flume-capture-events.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: page
+title: Capturing Events with Flume
+---
+
+## Purpose
```

Comment:
Readers who are not interested in Flume will not go to a page titled "Capturing Events with Flume."